src/components/Popup/Popup.js

```diff
@@ -31,7 +31,7 @@ const Popup = () => {
 
   return (
     <RSPopup
-      showHeader={false}
+      renderHeader={() => {}}
```

Comment:
Maybe we should set a default prop for this in react-spatial to avoid this?

Reply:
Then by default the Header will not be displayed